static/bidder-info/frvradn.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "https://fran.frvr.com/api/v1/openrtb"
+maintainer:
+  email: "engineering@frvr.com"
+gvlVendorID: 1107
+modifyingVastXmlAllowed: false
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+  site:
+    mediaTypes:
+      - banner
+      - video
+userSync:
+  iframe:
+    url: "https://fran.frvr.com/api/v1/sync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect_uri={{.RedirectURL}}"
+    userMacro: "{{UID}}"
```

Comment:
verified userSync is working 
<img width="1536" alt="image" src="https://user-images.githubusercontent.com/24757781/234246584-c92b988b-db29-4df3-94b0-e0cdb57ad4ca.png">

Reply:
verified. 👍